### PR TITLE
Gsettles/image atomics

### PR DESCRIPTION
**Pull request by gsettles01**
Task: task-[work-item-number]

Link: preview-305
[Explain the context of this pull request here]

## Testing

1. Step one
2. Step two
   Expected result:

## Additional information

[Optional]

### CODE REVIEW

File: css/src/atomics/image.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.object-fit-contain {
+  object-fit: contain;
+}
+.object-position-top {
```
Comment (wibjorn) [inline css/src/atomics/image.scss:4]: nit: new line

File: .changeset/slimy-pets-add.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
+---
+
+added image file including object-fit-contain and object-position-top
```
Comment (wibjorn) [inline .changeset/slimy-pets-add.md:5]: Looking for slightly clearer documentation of the change here.

Added .object-fit-contain and .object-position-top in new image related Atomics file (/src/atomics/image.scss).

**Comment by wibjorn** [comment]
We also need to add a documenation file (.md) in site/src/atomics/image.md. See other `atomics.template` for a template of how that page might look.

**Comment by gsettles01** [comment]
> We also need to add a documentation file (.md) in site/src/atomics/image.md. See other `atomics.template` for a template of how that page might look.

@wibjorn  added image.md in site

**Review by wibjorn — APPROVED** [review]

**Review by btelnes — APPROVED** [review]

**Pull request merged**